docs/source/caching/normalized-cache.mdx

```diff
@@ -232,16 +232,9 @@ We want to deduplicate entries like these by making sure they're assigned the _s
 
 Fortunately, all of our objects have an `id` field that we can use for this purpose. If an `id` is unique across _all_ objects in your graph, you can use its value directly as a cache ID. Otherwise if it's unique _per object type_, you can prefix it with the type name (as shown above).
 
-Note that `id` can be obtained from two sources:
```

Comment:
Removing this here because it looks like it'll have corresponding information in both of the individual ID articles